examples/heartbeat_demo/run_heartbeat_demo.sh

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bash
```

Comment:
Not sure if this is the best place for this script to live, since it runs the demo in a way that is intended mostly for testing purposes. What about putting this in `nucypher/scripts`? Another option is creating a scripts folder in the examples: `nucypher/examples/scripts`. We could put the fleet scripts there too.

Reply:
:+1:

Reply:
:heavy_check_mark: